3-networks/modules/standard_shared_vpc/variables.tf

```diff
@@ -14,41 +14,62 @@
  * limitations under the License.
  */
 
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 variable "project_id" {
   type        = string
-  description = "Project ID for Shared VPC."
+  description = "Project ID for Restricted Shared VPC."
 }
 
-variable "network_name" {
+variable "environment_code" {
   type        = string
-  description = "Name for VPC."
+  description = "A short form of the folder level resources (environment) within the Google Cloud organization."
 }
 
-variable "default_region" {
+variable "nat_region" {
   type        = string
-  description = "Default subnet region standard_shared_vpc currently only configures one region."
+  description = "Region used to create NAT cloud router."
 }
 
-variable "subnets" {
-  type        = list(map(string))
-  description = "The list of subnets being created"
-  default     = []
-}
-
-variable "secondary_ranges" {
-  type        = map(list(object({ range_name = string, ip_cidr_range = string })))
-  description = "Secondary ranges that will be used in some of the subnets"
-  default     = {}
+variable "vpc_label" {
+  type        = string
+  description = "Label for VPC."
 }
 
-variable "private_service_cidr" {
-  type        = string
-  description = "CIDR range for private service networking. Used for Cloud SQL and other managed services."
+variable "bgp_asn_nat" {
+  type        = number
+  description = "BGP ASN for NAT cloud routes."
 }
 
-variable "bgp_asn" {
-  type        = string
-  description = "BGP ASN for default cloud router."
+variable "subnets" {
```

Comment:
I think we should use the same format as the network module, so that people can define subnets as documented in that module

https://github.com/terraform-google-modules/terraform-google-network/blob/master/variables.tf#L37

Otherwise this will be an extra layer than needs to be maintained